Stanford/JumpstartEngineering/Install/Layouts/Sitewide.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @file
+ * Configure the sitewide layout.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Layouts;
+/**
+ *
+ */
+class Sitewide extends \AbstractInstallTask {
+
+  /**
+   * Configure the sitewide layout.
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
+
+
+    $context_status = variable_get('context_status', array());
```

Comment:
Yes, @cjwest knows why she did this:


JSE has a different sitewide layout than does JSA. This function sets the disabled status for JSA to "True" and the disabled status of JSE to "False."


________________________________
From: John Bickar <notifications@github.com>
Sent: Friday, March 25, 2016 7:41 AM
To: SU-SWS/stanford_install_tasks
Cc: Caryl J Westerberg
Subject: Re: [stanford_install_tasks] JSE! (#9)


In Stanford/JumpstartEngineering/Install/Layouts/Sitewide.php<https://github.com/SU-SWS/stanford_install_tasks/pull/9#discussion_r57449625>:

> +namespace Stanford\JumpstartEngineering\Install\Layouts;
> +/**
> + *
> + */
> +class Sitewide extends \AbstractInstallTask {
> +
> +  /**
> +   * Configure the sitewide layout.
> +   *
> +   * @param array $args
> +   *   Installation arguments.
> +   */
> +  public function execute(&$args = array()) {
> +
> +
> +    $context_status = variable_get('context_status', array());


No comment in the original install task. Maybe @cjwest<https://github.com/cjwest> knows why she wrote this: https://github.com/SU-SOE/stanford_sites_jumpstart_engineering/blob/7.x-1.x/includes/JumpstartSitesEngineering.php#L687.

-
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/SU-SWS/stanford_install_tasks/pull/9/files/bf4fa9f87de8e18c13327c1d857e43d00f8399e2#r57449625>